src/CommandingOfficers/AWBW/AWBWCommander.java

```diff
@@ -0,0 +1,71 @@
+package CommandingOfficers.AWBW;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderAbility;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.CommanderInfo.InfoPage;
+import Engine.GameScenario;
+import Engine.UnitMods.UnitDamageModifier;
+import Engine.UnitMods.UnitModifier;
+import Terrain.MapMaster;
+import Units.Unit;
+import Units.UnitDelta;
+
+public abstract class AWBWCommander extends Commander
```

Comment:
Seems odd that AWBW commander isn't based on TrilogyCommander, (or even AW2Commander) since that is the source material.
I guess this gets +10/10 like AW3, but gains charge like AW2? (I still don't understand the blurb on AW3 power charge).
The scaling also claims to go for 10 activations here, instead of the 9 for Trilogy.
Also things in the blurb are out of order compared to Trilogy, though it's probably hard to fully align across commanders due to how the blurbs are spread across the hierarchy.
So I guess uh, carry on, and it still surprises me how much nuance there is between the various AW implementations.